cmd/gaiacli/main.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/mint"
 	mintclient "github.com/cosmos/cosmos-sdk/x/mint/client"
 	mintrest "github.com/cosmos/cosmos-sdk/x/mint/client/rest"
+	distrcli "github.com/cosmos/cosmos-sdk/x/params/client/cli"
```

Comment:
This is imported already in the following line

Reply:
Thanks!